src/builder.rs

```diff
@@ -405,8 +443,10 @@ impl SaplingBuilder {
     pub fn build<SP: SpendProver, OP: OutputProver, R: RngCore, V: TryFrom<i64>>(
         self,
         mut rng: R,
+        padding_rule: &BundleType,
     ) -> Result<Option<(UnauthorizedBundle<V>, SaplingMetadata)>, Error> {
         let value_balance = self.try_value_balance()?;
+        let bundle_output_count = self.bundle_output_count(padding_rule)?;
```

Comment:
```suggestion
        bundle_type: &BundleType,
    ) -> Result<Option<(UnauthorizedBundle<V>, SaplingMetadata)>, Error> {
        let value_balance = self.try_value_balance()?;
        let bundle_output_count = self.bundle_output_count(bundle_type)?;
```
Also, should the bundle type be part of the builder configuration?

Reply:
That's how I originally designed this, but I think it's better to provide the bundle type as an argument to `build` because the bundle type does not otherwise influence the operation of the builder.